### PR TITLE
Timeline and color tweaks

### PR DESCRIPTION
**Pull request by silverwind**
Followup to https://github.com/go-gitea/gitea/pull/21784.

- Restore muted effect on timeline author and issuelist comment icon
- Remove whitespace inside shared user templates, fixing link hover underline
- Use shared author link template more
- Use `bold` class instead of CSS
- Fix grey-light color being too dark on arc-green
- Add missing black-light color
- Fix issuelist progress bar color
- Fix various other cases of missing `.muted`

Screenshot 2022-11-13 at 12 15 22
Screenshot 2022-11-13 at 12 16 52
Screenshot 2022-11-13 at 12 30 55
Screenshot 2022-11-13 at 12 41 03
Screenshot 2022-11-13 at 12 52 54
Screenshot 2022-11-13 at 12 56 16

### CODE REVIEW

**Comment by silverwind** [comment]
There might be more links we now need to add `muted` too, I will search them.

I plan to backport https://github.com/go-gitea/gitea/pull/21784 and this together to 1.18.

**Comment by silverwind** [comment]
More links fixed. There is one issue I had to workaround: When a `<a>` tag is part of the translation, I can not add a class to it without requiring re-translation. For such cases, I added a new `muted-links` class that. Example:

```
issues.ref_pull_from = `<a href="%[3]s">referenced this pull request %[4]s</a> <a id="%[1]s" href="#%[1]s">%[2]s</a>`
```

(Imho, we should remove all tags from translations, but that's another topic)

File: web_src/less/shared/issuelist.less [hunk 1]
```diff
@@ -91,6 +91,14 @@
           border-radius: 3px;
           vertical-align: 2px !important;
         }
+
+        progress::-webkit-progress-value {
+          background-color: var(--color-secondary-dark-5);
+        }
+
+        progress::-moz-progress-bar {
+          background-color: var(--color-secondary-dark-5);
```
Comment (delvh) [inline web_src/less/shared/issuelist.less]: Hmm, we could also think about using `color-small-accent` as that seems conceptually fitting.
What do you think?

**Review by delvh — APPROVED** [review]

File: web_src/less/shared/issuelist.less (hunk 1 above)
Comment (silverwind) [reply web_src/less/shared/issuelist.less]: I think it should be monochrome to fit the text around:

image

Thought it seems a bit too bright, I'll reduce by a shade.

Comment (silverwind) [inline web_src/less/shared/issuelist.less]:
```suggestion
          background-color: var(--color-secondary-dark-4);
        }

        progress::-moz-progress-bar {
          background-color: var(--color-secondary-dark-4);
```

**Review by lafriks — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Pull request merged**